### PR TITLE
Updated changelog.md

### PR DESCRIPTION
**Pull request by animeshdas2000**
Updated changelog as per the right order of release
image

### CODE REVIEW

File: sdk/template/azure-template/CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +16,17 @@
...
+
+
+## 1.0.0-beta.3 (2020-09-09)
+
+* (Engineering System) Enable master replacement link feature for cpp, attempt 2
+
+## 1.0.0-beta.2 (2020-09-09)
+
+* (Engineering System) Enable master replacement link feature for cpp, attempt 1
+
```
Comment (ahsonkhan) [inline sdk/template/azure-template/CHANGELOG.md:29]: This is duplicated below, remove.

```suggestion
```

File: sdk/template/azure-template/CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +16,17 @@
 
 * Validate vcpkg publishing automation framework
 
+## 1.0.0-beta.4 (Unreleased)
+
+
```
Comment (ahsonkhan) [inline sdk/template/azure-template/CHANGELOG.md:21]: This is duplicated above, remove.
```suggestion
```

**Review by ahsonkhan — COMMENTED** [review]
Thanks for fixing the version order. Please remove the two duplicate entries.

**Comment by ahsonkhan** [comment]
Fixes https://github.com/Azure/azure-sdk-for-cpp/issues/752

File: sdk/template/azure-template/CHANGELOG.md (hunk 2 above)
Comment (animeshdas2000) [reply sdk/template/azure-template/CHANGELOG.md]: Thank you for the suggestion, I have committed the changes.

**Review by ahsonkhan — APPROVED** [review]

**Pull request merged**